project/Docker.scala

```diff
@@ -0,0 +1,29 @@
+import com.typesafe.sbt.packager.Keys._
+import com.typesafe.sbt.packager.docker.DockerPlugin.autoImport.{Docker => docker}
+import sbt.Def
+import sbt.Keys._
+
+object Docker {
+
+  lazy val dockerSettings = Seq(
+    packageName in docker := packageName.value,
+    dockerBaseImage := "anapsix/alpine-java:8",
```

Comment:
Open question - do we want to have this as open-ended?  My instinct is that to ensure we can recreate images identically based only on the source code, we should set this explicitly (but update it regularly).  Obviously that's an overhead though.

Thoughts?